apps/www/lib/context/MeContext.tsx

```diff
@@ -42,7 +50,9 @@ css.global(
   },
 )
 
-type Me = {
+const AUTH_STATE_CHANGE_BROADCAST_STORAGE_KEY = 'authStateChangeBroadcast'
```

Comment:
couldn't we just listen to `ME_PORTRAIT_STORAGE_KEY`?

Reply:
Let's discuss this on the other comment